local_db/models.py

```diff
@@ -2,11 +2,38 @@
 from django.utils import timezone
 from ulid import ulid
 
+from airlock.api import Status
+
 
 def local_request_id():
     return str(ulid())
 
 
+class StatusField(models.TextField):
```

Comment:
Do we want to inherit from CharField rather than TextField?

Reply:
No, I didn't have a particularly strong opinion, except that in a normal django model I'd usually use a CharField for short text fields (but only because that's how I learned to write django fields). If you have a rationale for choosing TextField, that's good for me.